docker-compose.yml

```diff
@@ -0,0 +1,19 @@
+version: '3.6'
+volumes:
+  dbdata:
+services:
+  db:
+    image: postgres:9.6-alpine
+    volumes:
+      - dbdata:/var/lib/postgresql/data
+  web:
+    build: .
+    image: ${dockerHubUsername}/${dockerHubImageName}:latest
```

Comment:
If I understand right, out of the box this breaks local dev. Should we use a `docker-compose.development.yml`? 

I'm not sure what good practice is here. Dev/prod parity seems good - interacting with DockerHub from dev definitely feels like a step too far.

Reply:
Yes, it does break local dev. I have similar feelings about parity. Happy not to support local dev at all in this branch and maybe do that next?

Reply:
Agreed, but I'd probably change the README to reflect that if so.

Reply:
Very good point 😄

Reply:
I changed it. It's a bit uncompromising now — you must specify chromedriver — but that's the world we live in on this branch. No objections if you want to make it a *bit* dev-friendlier before merging (I'm not in tmrw)

Reply:
I'm quite happy to move forward as-is, the chromedriver install isn't too arduous. We may not get this merged tomorrow as I think @tpinney doesn't work Fridays either, and I'd like to double-check he's all happy too (unless you've already had that conversation?)

Also we're far past home-time on this very hot day - please don't feel obliged to respond this week! :sunny: